### ISSUE TITLE
scheduled_tasks not executing

### ISSUE THREAD

**Issue by ayjaweems** [drajer-health/eCRNow#640]
While testing the v3 version of the eCRNow app, requests to the _/launchPatient_ endpoint are successful but when querying the database there are `scheduled_tasks` entries with no values for the `last_failure`, `last_success,` and `last_heartbeat` columns when the `execution_time` has passed. There are also no entries in the `ph_messages` table. The logs didn't reveal any errors besides a few cases of valid not found errors where the FHIR data wasn't available yet. When the app was launched again for those & other records with valid FHIR data, I wasn't able to find any failures in the logs.

**Comment by mmatiasalt** [comment]
Hi @ayjaweems , 
Thank you for your inquiry. To better assist you, can you please advise which EHR you are using and please upload Loading Queries/XML or any logs showing that example? Please also include screenshots if applicable. Thank you .

**Comment by nbashyam** [comment]
Can you let us know what your Encounter.period values are and Encounter.status values ?

**Comment by ayjaweems** [comment]
@nbashyam The Encounter.period is 
```
period:
  start: '2024-01-24T12:00:00+00:00'
```
and the Encounter.status is `finished`

**Comment by nbashyam** [comment]
I think we discussed this and you are good to go, please confirm.

**Comment by ayjaweems** [comment]
> The error is related to your Direct Transport and the HISP, are you able to send data via the HISP manually. Also what HISP are you using ?

@nbashyam I've switched to sending the eICR data using the Rest API endpoint and is working successfully. I also realized the files weren't being logged to the `bsa.output.directory` path because the `save.debug.files` was false in the configuration so we're getting debug files outputted now in that folder. Now the email logic is working successfully.

The `ph_messages` table is populated now a few minutes after the patient is launched. How long does it take for the reportability response document to be received? We're using a Rest API in that case as well. I see the scheduled task created for the encounter has an execution time of three days from when the app was launched, would the RR document be generated and sent back after that time?
